.github/workflows/build.yml

```diff
@@ -14,37 +14,26 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
 
     name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
 
     strategy:
       fail-fast: false
       matrix:
-        php: [8.0]
-        symfony: [^5.4, ^6.0]
-        sylius: [~1.11.0, ~1.12.0]
-        node: [14.19]
-        mysql: [5.7, 8.0]
-
-        exclude:
-          -
-            sylius: ~1.11.0
-            symfony: ^6.0
-          -
-            sylius: ~1.11.0
-            php: 7.3
-          -
-            sylius: ~1.11.0
-            php: 7.4
+        php: ["8.1"]
```

Comment:
Sylius 1.12 has ^8.0 php rule.

Reply:
Sure. I didn't know that I can exclude some matrix parameters in build process.